pkg/providers/hcl/services.go

```diff
@@ -37,6 +36,11 @@ func ParseIntermediateService(parent *broker.Context, manifest Service) (*specs.
 		return nil, err
 	}
 
+	resolver := "default"
```

Comment:
should we call the resolver default or dns? I think DNS describes the way on how addresses will be resolved a bit more.

Reply:
probably yes, renamed.